### ISSUE TITLE
Catch and report all no RTTI data errors

### ISSUE THREAD

**Issue by macumber** [NREL/OpenStudio#1671]
No RTTI data exceptions seem to be thrown for several types of errors.  We should catch these (e.g. in the top level application error handling) and display a more informative error to the user.  Something like:  An unknown error has occurred, please send your model to OpenStudio@nrel.gov for diagnostics.

**Comment by jmarrec** [comment]
+1

**Comment by jmarrec** [comment]
it's been a while, but it's back. When I click on the Measures tab in OS App (1.11.0)

![no rtti data](https://cloud.githubusercontent.com/assets/5479063/14847596/c60bde60-0c69-11e6-9707-85714e4cd5f4.png)

----

To be transparent I know what I did wrong in this particular case. As stated in #2206 I had an orphan node problem, so I did this loop:

```ruby
model.getNodes.each do |node|
  if node.isRemovable
    puts node.name
    node.remove
  end
end
```

This deleted the node I wanted gone, but also deleted some other "removable" nodes that really couldn't be removed. After investigation, these were the thermal zones' Zone Air Nodes, and it's saying it's removable because there's only an inlet port defined, no outlet.
